src/pages/EditUserPage.ts

```diff
@@ -0,0 +1,94 @@
+import { expect } from '@/fixtures/fixtures';
+import BasePageModel from '@/pages/BasePageModel';
+
+class EditUserPage extends BasePageModel {
+  async isLoaded() {
+    await expect(this.page.locator('title')).toBeVisible();
+  }
+
+  get userTitle() {
+    return this.page.locator('.summary .title');
+  }
```

Comment:
```suggestion
  get summary() {
    return this.page.role('region', { name: 'summary' });
  }
```

the in the test just check for
```ts
await expect(editPage.summary).toContainText(`name lastname`)
```